core/runtime.rs

```diff
@@ -3737,21 +3737,6 @@ assertEquals(1, notify_return_value);
     })
   }
 
-  #[test]
-  fn test_core_js_stack_frame() {
-    let mut runtime = JsRuntime::new(RuntimeOptions::default());
-    // Call non-existent op so we get error from `core.js`
-    let error = runtime
-      .execute_script_static(
-        "core_js_stack_frame.js",
-        "Deno.core.opAsync('non_existent');",
```

Comment:
These are async ops and the error for non-existent ops was made async as well. opAsync and opAsync2 should be going away soon, so all of these tests will eventually go away too.